### ISSUE TITLE
Upgrading to 3.0.61

### ISSUE THREAD

**Issue by Kressilac** [Microsoft/slow-cheetah#49]
When I previewed an app.config in my console application, SlowCheetah complained that I needed to upgrade.  I was running 2.5.48.  When I installed 3.0.61, it added EnsureNuGetPackageBuildImports to the bottom of my file along with an Import Project node.  Is this expected?  I thought you had to remove this stuff to use NuGet 3.0?

**Comment by davilimap** [comment]
This is not exactly a SlowCheetah behaviour. It installs itself using the NuGet API in Visual Studio, so any changes to the project file are handled by that. The Import Project node is needed by SlowCheetah to add the necessary MSBuild targets to your project so the transforms are executed on build. The `EnsureNuGetPackageBuildImports` target is a NuGet behaviour for .NET Framework projects. As far as I know, you don't need to remove any of this to use NuGet.

**Comment by davilimap** [comment]
Could you clarify what you mean by needing to remove this stuff? If you mean that it should be removed for the newest SlowCheetah to work, that is actually another property that is removed automatically by the newest version of the extension (3.0.61). If you mean that it needs to be removed for NuGet to work, as far as I know it's added by NuGet itself, so it should be fine.

**Comment by Kressilac** [comment]
I may have read the nugget team's upgrade guidance wrong.  It stated to remove references to nuget.targets from the ensure block of code.  I took it to mean remove the whole section from the csproj file.  Until we installed, slow cheetah, this was one and the same because there were no other references in the project file within the ensure block.  Seems I misread the guidance from nugget.

Hope that clarifies things.



Derek Licciardi 502 552-5030


-------- Original message --------
